src/Laravel/TermwindServiceProvider.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Termwind\Laravel;
+
+use Illuminate\Support\ServiceProvider;
+use Termwind\Termwind;
+
+final class TermwindServiceProvider extends ServiceProvider
+{
+    /**
+     * Sets the the renderer used for unit testing.
+     */
+    public function register(): void
+    {
+        if ($this->app->runningUnitTests()) {
```

Comment:
Codewise is perfect. Yet, I think you create a DeferableOutput class that should be used like this:

```php
DeferableOutput::from(fn () => app()->make(...
```

What do you think?